src/Wallabag/CoreBundle/Repository/EntryRepository.php

```diff
@@ -86,6 +86,25 @@ public function getBuilderForStarredByUser($userId)
     }
 
     /**
+     * Retrieves entries filtered with a search term for a user.
+     *
+     * @param int    $userId
+     * @param string $term
+     *
+     * @return QueryBuilder
+     */
+    public function getBuilderForSearchByUser($userId, $term)
+    {
+        return $this
+            ->getBuilderByUser($userId)
+            ->andWhere('e.content LIKE :term')->setParameter('term', '%'.$term.'%')
+            ->orWhere('e.title LIKE :term')->setParameter('term', '%'.$term.'%')
+            ->leftJoin('e.tags', 't')
+            ->groupBy('e.id')
```

Comment:
@j0k3r what about `getBuilderForUntaggedByUser` where we already have this `groupBy`?